### PR TITLE
Ami creation

### PR DESCRIPTION
**Pull request by muhammadIkramUlHaq**

### CODE REVIEW

**Comment by rasheedamir** [comment]
@hazim1093 plz review this PR and merge it!

File: Makefile [hunk 1]
```diff
@@ -83,6 +84,9 @@ destroy_all: \
 	destroy_gocd \
 	destroy_elk \
 	destroy_worker \
+	destroy_application \
+	destroy_snapshot \
+	destroy_amicreation \
```
Comment (rasheedamir) [inline Makefile]: @hazim1093 how about renaming the module to "amicreator" or "animator" or "baker" etc.

**Comment by rasheedamir** [comment]
@hazim1093 plz add some description to the new added modules which explains what it does and what is the purpose of it

**Comment by hazim1093** [comment]
@rasheedamir 
Renamed the modules and added readme files.

**Pull request merged**